example/v2/thinevent_webhook_handler/thinevent_webhook_handler.go

```diff
@@ -0,0 +1,95 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"io"
+	"net/http"
+	"os"
+
+	"github.com/stripe/stripe-go/v80"
+	billingMeters "github.com/stripe/stripe-go/v80/billing/meter"
+	"github.com/stripe/stripe-go/v80/rawrequest"
+	webhook "github.com/stripe/stripe-go/v80/webhook"
+)
+
+var apiKey = "{{API_KEY}}"
+var webhookSecret = "{{WEBHOOK_SECRET}}"
+
+func main() {
+	b, err := stripe.GetRawRequestBackend(stripe.APIBackend)
+	if err != nil {
+		fmt.Println(err)
+		os.Exit(1)
+	}
+
+	client := rawrequest.Client{B: b, Key: apiKey}
```

Comment:
From what I understood, we introduce the client in the rawrequest package so that we can use a different backend than the default. So I get why we want to use it in the `meter_event_stream.go` example, but not why we would want to use it here

Reply:
I think there are a couple reasons to want to do this in the example here:

1. using rawrequest.Client here keeps the interface consistent across all raw request examples, and I think is what we want users to do longer term, because
2. this feels more akin to the StripeClient pattern but is also structured the same as other golang services.  so I think migration in the future will be simpler if we introduce StripeClient but it's not going to feel out of place in the mean timer

This change is also coupled with a change in rawrequest/client.go which removes Get, Post, and Delete.  My thought here was just retired the preview API (e.g. `Get`, `Post`, `Delete`) in other SDKs in lieu of a single `RawRequest`, so it additionally keeps things consistent with our other SDKs.